src/worker/util/defaultGameAttributes.ts

```diff
@@ -137,6 +137,7 @@ const defaultGameAttributes: GameAttributesLeagueWithHistory = {
 	numActiveTeams: 0,
 	difficulty: 0, // See constants.DIFFICULTY for values
 	tradeDeadline: 0.6,
+	goatFormula: wrap(""),
```

Comment:
Change `wrap("")` to just `""`. `wrap` is what enables storing history of a setting, which is needed for pointsFormula to correctly display historical standings, but is not needed for goatFormula.